<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1290</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="modeTabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1141</width>
      <height>761</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="dream">
     <attribute name="title">
      <string>Dream</string>
     </attribute>
     <widget class="BaseView" name="dw">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>831</width>
        <height>711</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>10</y>
        <width>281</width>
        <height>30</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QFrame" name="modelFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="modelColorButton">
         <property name="text">
          <string>Цвет детали</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="addButton">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>80</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить деталь</string>
      </property>
     </widget>
     <widget class="QComboBox" name="typeChooseBox">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>50</y>
        <width>281</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="educate">
     <attribute name="title">
      <string>Educate</string>
     </attribute>
     <widget class="BaseView" name="ew">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>831</width>
        <height>711</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>10</y>
        <width>281</width>
        <height>30</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QFrame" name="modelEdFrame">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="modelEdColorButton">
         <property name="text">
          <string>Цвет детали</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QComboBox" name="typeEdChooseBox">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>50</y>
        <width>281</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addEdButton">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>80</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Добавить деталь</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableEdWorkLimit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>180</y>
        <width>281</width>
        <height>541</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Деталь</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Количество</string>
       </property>
      </column>
     </widget>
     <widget class="QTableWidget" name="tableEdDetailLimit">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>70</y>
        <width>281</width>
        <height>621</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Деталь</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Количество</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="closeTableButton">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Закрыть таблицу</string>
      </property>
     </widget>
     <widget class="QPushButton" name="openTableButton">
      <property name="geometry">
       <rect>
        <x>850</x>
        <y>130</y>
        <width>281</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Задать новые значения</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1290</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BaseView</class>
   <extends>QOpenGLWidget</extends>
   <header>baseview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
